src/components/book.js

```diff
@@ -18,7 +18,7 @@ function Book({ book, deleteBook }) {
           <p>Category:</p>
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍